featuretools/primitives/utils.py

```diff
@@ -398,179 +397,6 @@ def _find_class_in_descendants(self, search_key):
                 return cls
 
 
-def _roll_series_with_gap(series, window_size, gap=0, min_periods=1):
```

Comment:
Yeah I would like that as well. It would just be inconsistent with how the repo is currently structured since all agg and trans primitives are just listed in a single file. I'd be down with this if we did this for the rest of the primitives. I was going to take it a step further and just put these util methods in a utils class where they are accessible as staticmethods but I might be going too far on the OOP side of the spectrum lol.

Reply:
Naw, you get basically the same namespacing concept you want by leveraging python modules.